### ISSUE TITLE
List Sorting direction=desc doesn't work and is the same as direction=asc

### ISSUE THREAD

**Issue by ckjpn** [Tatoeba/tatoeba2#1803]
Before clicking the "date added to list" a second time reversed the sort.
Now that doesn't happen.

Also, these URLs now get the same sort.

https://tatoeba.org/eng/sentences_lists/show/8760/und?sort=created&direction=desc
https://tatoeba.org/eng/sentences_lists/show/8760/und?sort=created&direction=asc

**Comment by AndiPersti** [comment]
@ckjpn Is this still a problem for you?

I've tried to reproduce this issue, but both your links (and others I've tried) work for me, i.e. they show the sentences in descending and ascending order respectively. Also clicking on either "date added to list" and "date created" will reverse the current ordering.

(I couldn't find any commits since February 28th which would explain the fix for this issue but I've probably looked at the wrong places.)
